src/components/cards/CardsContainer.svelte

```diff
@@ -0,0 +1,28 @@
+<script>
+  import {
+    CARD_DEFAULT_WIDTH,
+    CARD_CONTAINER_DEFAULT_GAP,
+  } from "./Card.constants";
+
+  export let gridGap = CARD_CONTAINER_DEFAULT_GAP;
+  export let cardWidth = CARD_DEFAULT_WIDTH;
+</script>
+
+<style>
+  ul {
+    width: 100%;
+    display: grid;
```

Comment:
I love CSS grid. One of my reasons for pushing for the redesign was to drop support for IE11 and make more use of all the new js and css features that get shipped in modern browsers.

Reply:
Makes sense, supporting IE can be pretty painful at times.